explorer/src/components/Domain/DomainBlockTimeProgress.tsx

```diff
@@ -0,0 +1,77 @@
+import { Spinner } from 'components/common/Spinner'
+import { NotFound } from 'components/layout/NotFound'
+import {
+  DomainsStatusQuery,
+  DomainsStatusQueryVariables,
+  Order_By as OrderBy,
+} from 'gql/types/staking'
+import { useSquidQuery } from 'hooks/useSquidQuery'
+import { FC, useMemo } from 'react'
+import { capitalizeFirstLetter } from 'utils/string'
+import { QUERY_DOMAIN_STATUS } from './staking.query'
+
+export const DomainBlockTimeProgress: FC = () => {
+  const { data, loading, error } = useSquidQuery<DomainsStatusQuery, DomainsStatusQueryVariables>(
+    QUERY_DOMAIN_STATUS,
+    {
+      variables: {
+        limit: 10,
+        orderBy: [{ id: OrderBy.Asc }],
+        where: {},
+      },
+      pollInterval: 2000,
+      context: { clientName: 'staking' },
+    },
+  )
+
+  const domainData = useMemo(() => {
+    if (loading || error || !data) return []
+
+    return data.domain.map((domain) => {
+      return {
+        domain: domain.name,
+        stats: [
+          { title: 'Last 1 Epoch', intervalSeconds: 2 },
+          { title: 'Last 6 Epochs', intervalSeconds: 2 },
+          { title: 'Last 144 Epochs', intervalSeconds: 2 },
+          { title: 'Last 1008 Epochs', intervalSeconds: 2 },
+        ],
+      }
+    })
+  }, [data, loading, error])
```

Comment:
Do we want to merge this with static data?

Reply:
This feature was relying on upcoming changes (and future fixes) to the staking-indexer, the changes have been made, but this new indexer is not sync yet
![image](https://github.com/user-attachments/assets/e3bcf93e-caaa-4aa5-95ff-bbb171dc8732)